shared-ui/src/main/java/org/hzontal/shared_ui/switches/TellaSwitchWithMessage.kt

```diff
@@ -93,6 +98,16 @@ class TellaSwitchWithMessage @JvmOverloads constructor(
         }
     }
 
+    fun isDisabledTheme(isDisabled: Boolean) {
```

Comment:
maybe we could change the method name "isDisabledTheme" to "setSwitchTrackColor" to better reflect its purpose.